src/constants/textConstants.js

```diff
@@ -1,13 +1,55 @@
 import React from 'react';
+import PropTypes from 'prop-types';
+
 import keyMirror from 'keymirror';
+import { helpers } from '../utils/';
 
 const TEXT_TAGS = keyMirror({
+  condensed: null,
   span: null,
   p: null,
   strong: null
 });
 
+const CondensedText = ({
```

Comment:
Sorry to be annoying, especially after all the discussion around merging this with Text, but this feels out of place in a `textConstants` file. Maybe this component can be exported from the Text directory and imported here? That does feel a little tangled though... 🤔 Thoughts @tynes?

Reply:
`CondensedText` is definitely more of a component than a constant right?